projects/media-viewer/src/lib/bookmark/services/bookmark-api.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+// @todo check if I can juts ref the BookmarkApiService in the annotations module instead?
+@Injectable()
+export class BookmarkApiService {
```

Comment:
there's already a bookmarks-api service that has this endpoint
https://github.com/hmcts/media-viewer/blob/master/projects/media-viewer/src/lib/annotations/bookmarks-api.service.ts